src/core/network.cpp

```diff
@@ -43,6 +44,20 @@ Network::Network( geojson::GeoJSON fabric ){
     else{
       v1 = this->descriptor_map[ feature_id];
     }
+
+    if ( this->level_map.find(feature_id) == this->level_map.end() )
+    {
+      if ( feature->has_property("level") )
+      {
+        const auto& prop = feature->get_property("level");
```

Comment:
This again doesn't match the data/catchment_data_multilayer.geojson file @donaldwj